examples/prompty/basic/prompty-quickstart.ipynb

```diff
@@ -79,78 +79,105 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from promptflow.core import Prompty, AzureOpenAIModelConfiguration, OpenAIModelConfiguration\n",
+    "from promptflow.core import Prompty\n",
     "\n",
     "# load prompty as a flow\n",
     "f = Prompty.load(source=\"basic.prompty\")\n",
     "\n",
+    "# execute the flow as function\n",
+    "result = f(\n",
+    "    first_name=\"John\", last_name=\"Doe\", question=\"What is the capital of France?\"\n",
+    ")\n",
+    "result"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Or you can override model configuration of prompty with dict."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "# override model configuration of prompty with dict\n",
+    "override_model = {\n",
+    "    \"configuration\": {\n",
+    "        \"type\": \"azure_openai\",\n",
+    "        \"azure_deployment\": \"gpt-35-turbo\",\n",
+    "        \"api_key\": \"${env:AZURE_OPENAI_API_KEY}\",\n",
```

Comment:
add comment say user can point api_key to other env variable

Reply:
fix it